challenger-ng/src/app/store/actions/photo.actions.spec.ts

```diff
@@ -0,0 +1,7 @@
+import * as fromPhoto from './photo.actions';
```

Comment:
The tests are failing... This will fix it:

`describe('loadPhoto', () => {
  it('should return an action', () => {
    expect(fromPhoto.loadPhoto().type).toBe('[Photo] Load Photo');
  });
});
`

Reply:
Updated